src/containers/Bookings.js

```diff
@@ -2,22 +2,37 @@
 import React, { Component } from "react";
 import Search from "../components/search/Search.js";
 import Results from "../components/results/Results.js";
-import * as FakeBookings from "../data/fakeBookings.json";
+
+// import * as FakeBookings from "../data/fakeBookings.json";
 
 export default class Bookings extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      FakeBookings: '',
-      term: '',
+      FakeBookings: [],
+      term: null,
       result: false,
       direction: {
-        sortin: 'asc'
+        sortin: 'asc',
+        error: null,
```

Comment:
I wouldn't put the `error` state within `direction` state. Those aren't really related so I'd split them apart